pkg/config/validation.go

```diff
@@ -91,3 +91,35 @@ func ValidateServerOptions(options ServerOptions) error {
 
 	return nil
 }
+
+func ValidatePruneOptions(options PruneOptions) error {
+	missingSet := make(map[string]struct{})
+	customSet := make(map[string]struct{})
```

Comment:
Do you plan on using customSet? Otherwise it can be deleted.

Reply:
I shall delete